### DISCUSSION TITLE
What is V_A in the draw

### DISCUSSION THREAD

**Discussion by BabelDev0** [quantum-engineering-polito/CourseNotes#25] [QD] [answered]
@deskclam what is $V_A$ in the draw?
<img width="962" alt="Screenshot 2024-01-11 at 20 22 34" src="https://github.com/quantum-engineering-polito/CourseNotes/assets/91274142/0cecafcf-c9c8-43b0-ad57-fcfc5d8bcdec">

**Comment by deskclam** [comment, marked as answer]
@BabelDev0 Is the voltage applied between bulk and the two wells, I already changed the note and explained it
